tests/agents/arbitrage_agent/test_arbitrage_agent.py

```
@@ -0,0 +1,64 @@
+    m1.question = "Will Kamala Harris become the US president in 2025?"
+    yield m1
+
+
+@pytest.fixture(scope="module")
+def unrelated_market() -> t.Generator[AgentMarket, None, None]:
+    m1 = Mock(OmenAgentMarket, wraps=OmenAgentMarket)
+    m1.question = "Will Donald Duck ever retire from his adventures in Duckburg?"
+    yield m1
+
+
+@pytest.mark.skipif(not RUN_PAID_TESTS, reason="This test costs money to run.")
```

Ruff

Undefined name `RUN_PAID_TESTS`

(F821)